script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -277,7 +277,7 @@ I'm sweating so much.
 [sha:c675989ae49f7e6b0c938ceb117cf675626c8e71]{
 -- Page 206, Offset 4507.
 -- @g　<生態|じしん>が<動物|いきもの>である事を思い出す。
-//The double meaning here is intentional, showing him "regressing" to a beast
+// The double meaning here is intentional, showing him "regressing" to a beast
```

Comment:
Is there any practical reason you do this, or is just for neatness sake?

Reply:
Neatness sake. I don't do it manually, the reinjection process automatically reformats all the comments.